lcd.cpp

```diff
@@ -802,12 +826,17 @@ void LCD_update() {
     do {
     #endif
       (*current_menu)();
-    #if LCD_TYPE == LCD_IS_128X64
+
+      // reset.  must come after first (*current_menu)() because it might be used there, 
+      // and it should only be used once there if nextPage makes this loop.  (damn globals...)
+      lcd_turn -= lcd_turns_to_process; 
```

Comment:
this is now inside a loop.  was outside.  might be run more times than intended.

Reply:
It needs to be inside the loop so the value isn't used multiple times inside of "current_menu".  That was a bug in the original code.  The lcd_turns_to_process value is set to 0, so that current_menu only uses a non-zero lcd_turns_to_process once.  And have to adjust lcd_turn before lcd_turns_to_process is zeroed out.